ext/curl/curl_string_file.stub.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @generate-function-entries
+ * @generate-class-entries
+ */
+
+class CURLStringFile
+{
+    public string $data = "";
+    public string $postname = "";
```

Comment:
You can leave off the default values on these properties (and on $mime as well if you assign it unconditionally).